guide/driver-management/driver-management.bal

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+import ballerina/jms;
+
+type Person record {
+    string name;
+    string address;
+    string phonenumber;
+    string registerID;
+    string email;
+};
+
+endpoint http:Listener listener {
+    port:9091
+};
+
+// Initialize a JMS connection with the provider
+// 'Apache ActiveMQ' has been used as the message broker
+jms:Connection conn = new({
+        initialContextFactory: "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+        providerUrl: "tcp://localhost:61616"
+    });
+
+// Initialize a JMS session on top of the created connection
+jms:Session jmsSession = new(conn, {
+        // Optional property. Defaults to AUTO_ACKNOWLEDGE
+        acknowledgementMode: "AUTO_ACKNOWLEDGE"
+    });
+
+// Initialize a queue receiver using the created session
+endpoint jms:QueueReceiver jmsConsumer {
+    session:jmsSession,
+    queueName:"trip-driver-notify"
+};
+
```

Comment:
Do we need this much of new lines here?? I think having a new line would be enough between different constructs. WDYT?